resources/sts/4.14/hypershift/sts_extended_hcp_support_permission_policy.json

```diff
@@ -0,0 +1,115 @@
+                "ec2:DescribeLocalGatewayRouteTableVpcAssociations",
+                "ec2:DescribeLocalGateways",
+                "ec2:DescribeLocalGatewayVirtualInterfaceGroups",
+                "ec2:DescribeLocalGatewayVirtualInterfaces",
+                "ec2:DescribeManagedPrefixLists",
+                "ec2:DescribeNetworkInsightsAnalyses",
+                "ec2:DescribeNetworkInsightsPaths",
+                "ec2:DescribePlacementGroups",
+                "ec2:DescribePrefixLists",
+                "ec2:DescribePrincipalIdFormat",
+                "ec2:DescribePublicIpv4Pools",
+                "ec2:DescribeSnapshotAttribute",
+                "ec2:DescribeSnapshots",
+                "ec2:DescribeSpotFleetInstances",
+                "ec2:DescribeTags",
+                "ec2:DescribeTransitGatewayAttachments",
+                "ec2:DescribeTransitGatewayConnectPeers",
+                "ec2:DescribeTransitGatewayConnects",
+                "ec2:DescribeTransitGatewayMulticastDomains",
+                "ec2:DescribeTransitGatewayPeeringAttachments",
+                "ec2:DescribeTransitGatewayRouteTables",
+                "ec2:DescribeTransitGateways",
+                "ec2:DescribeTransitGatewayVpcAttachments",
+                "ec2:DescribeVpcAttribute",
+                "ec2:DescribeVpcClassicLink",
+                "ec2:DescribeVpcClassicLinkDnsSupport",
+                "ec2:DescribeVpcEndpointConnectionNotifications",
+                "ec2:DescribeVpcEndpointServiceConfigurations",
+                "ec2:DescribeVpcEndpointServicePermissions",
+                "ec2:DescribeVpcEndpointServices",
+                "ec2:DescribeVpcPeeringConnections",
+                "ec2:DescribeVpnConnections",
+                "ec2:DescribeVpnGateways",
+                "ec2:GetAssociatedIpv6PoolCidrs",
+                "ec2:GetManagedPrefixListEntries",
+                "ec2:GetSerialConsoleAccessStatus",
+                "ec2:GetTransitGatewayAttachmentPropagations",
+                "ec2:GetTransitGatewayMulticastDomainAssociations",
+                "ec2:GetTransitGatewayPrefixListReferences",
+                "ec2:GetTransitGatewayRouteTableAssociations",
+                "ec2:GetTransitGatewayRouteTablePropagations",
+                "ec2:ModifyInstanceAttribute",
+                "ec2:SearchLocalGatewayRoutes",
+                "ec2:SearchTransitGatewayMulticastGroups",
+                "ec2:SearchTransitGatewayRoutes",
+                "ec2:StartNetworkInsightsAnalysis",
+                "elasticloadbalancing:ConfigureHealthCheck",
+                "kms:CreateGrant",
+                "s3:GetBucketTagging",
+                "s3:GetObjectAcl",
+                "s3:GetObjectTagging",
+                "s3:ListAllMyBuckets",
+                "tiros:CreateQuery",
+                "tiros:GetQueryAnswer",
+                "tiros:GetQueryExplanation",
+                "ec2:RunInstances",
```

Comment:
I am not seeing the "TerminateInstances" perms to run the osd-network-verifier, are you removing that on purpose?

Reply:
Ignore above, I see it's already in the default support policy.